pages/resources/anonymousquestions.vue

```diff
@@ -0,0 +1,62 @@
+<script>
+/* eslint-disable */
+</script>
+
+<style>
+.content {
+  max-width: 500px;
+  margin: auto;
+}
+</style>
+ 
+<template>
+  <div class="content">
+    <h1 style="font-size: 50px;"> Ask an Upper Year! </h1>
+    <!-- <form action="https://formspree.io/xwkrdzyg" method="POST">-->
+    	<form action="http://individual.utoronto.ca/cgi-bin/formmail.pl" method="POST">
+	<input type="hidden" name="recipient" value="zain.kazmi@mail.utoronto.ca"/>
+	<li style="list-style-type: none;"><label for="ask_alex">Ask Alex: TA and POST</label>
+      	<input type="checkbox" name="ask alex" text="Ask Alex" id="ask_alex"/>
+      	</li>
+      	<li style="list-style-type: none;">
+      	<label for="ask_jess">Ask Jess</label>
+     	<input type="checkbox" name="Ask Jess" id="ask_jess"/>
+      	</li>
+      	<li style="list-style-type: none;">
+     	<label for="ask_chris">Ask Chris</label>
+      	<input type="checkbox" id="ask_chris" name="Ask Chris" />
+      	</li>
+	<li style="list-style-type: none;">
+      	<label for=Question> Input your Question here </label> 
+      	</li>
+	<li style="list-style-type: none;">
+	<textarea id="Question" name="Question" style="border-style: inset;"></textarea>
```

Comment:
![image](https://user-images.githubusercontent.com/19757203/92273154-de592a00-eeb8-11ea-9e7b-6ceacdf74e5a.png)

The border was invisible here.